docs/Content structure/Paragraphs/Group Schedules.md

```diff
@@ -0,0 +1,9 @@
+# Group Schedules
+
+This is dynamic paragraph that renders the group schedules from GroupEx Pro.
+
+### Fields
+
+| Name  | Machine name | Required | Description |
+| ------------- | ------------- | ------------- | ------------- |
+| Block | field\_prgf_block | Yes | Block reference to the view/block. Should have default value and is configurable in form display. |
```

Comment:
modules/custom/openy_group_schedules/assets/endorsed_by_silverfit_sm.png file down below could ve removed, because not used